src/nav/AppNavigator.js

```diff
@@ -59,7 +59,14 @@ export const createAppNavigator = (args: {|
       'account-details': { screen: AccountDetailsScreen },
       'group-details': { screen: GroupDetailsScreen },
       auth: { screen: AuthScreen },
-      chat: { screen: ChatScreen },
+      chat: { screen: ChatScreen,
+      navigationOptions:{
+        gestureEnabled:true,
+        gestureDirection:'horizontal',
+        gestureResponseDistance:{
+          horizontal:20
```

Comment:
How did you arrive at the number 20 for `gestureResponseDistance.horizontal`? 🙂 The [doc](https://reactnavigation.org/docs/4.x/stack-navigator/#gestureresponsedistance) says it defaults to 25. If we're not keeping the default, let's write down the reason in a code comment.